scala/scala.bzl

```diff
@@ -786,6 +803,8 @@ def scala_repositories():
 
   native.bind(name = "io_bazel_rules_scala/dependency/scalatest/scalatest", actual = "@scalatest//jar")
 
+# With providers changes, this is completely deprecatable once bazel release includes
+# fixes to java rules using JavaProvider
```

Comment:
Sadly, not yet, exports was what was specifically broken and fixed by:

https://github.com/bazelbuild/bazel/commit/7e91ad631a660ae12a2b8dcfb8ba42574a7a76d3

However, might delay submitting this PR until that commit is released, in which case will definitely do that.